include/cutlass/arch/barrier.h

```diff
@@ -36,7 +36,11 @@
 
 #include <cutlass/arch/memory_sm75.h>
 #include <cute/arch/cluster_sm90.hpp>
-#if defined(__CUDA_ARCH__) && __CUDA_ARCH__ >= 900 && (__CUDACC_VER_MAJOR__ >= 12)
+
+#if defined SYCL_INTEL_TARGET
+SYCL_EXTERNAL __attribute__((convergent)) void __spirv_ControlBarrierWaitINTEL(int, int, int);
```

Comment:
please name args

Reply:
Named the args following this: https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#OpControlBarrierArriveINTEL